### ISSUE TITLE
inference.ipynb audio output silent

### ISSUE THREAD

**Issue by abaddon-moriarty** [NVIDIA/tacotron2#475]
Hi, I'm trying to run the inference.ipynb with the published models, neither model got errors.
![image](https://user-images.githubusercontent.com/56978503/115398207-35610480-a1e7-11eb-9f32-9041cbb6567d.png)
Also there are no problems with the plot results:
![image](https://user-images.githubusercontent.com/56978503/115398533-8a047f80-a1e7-11eb-8c2c-6430e753f81b.png)
But when I try to play the audio output, there is no voice, my speakers are definitely detecting a sound but it's just white noise, and yet again there are no explicit errors.
![image](https://user-images.githubusercontent.com/56978503/115398687-b6200080-a1e7-11eb-88a7-a3c6e2d81274.png)
I'm very new to voice synthesis, there might be something I overlooked, but I followed the indications to the t, so I'm not sure what could have gone wrong.

I have looked through other issues but I haven't seen any problem similar to this, if I have missed it, thank you for redirecting me to those issues.
any help would be appreciated.

**Comment by holynova-SD** [comment]
I also have this problem, have you solved it？

**Comment by abaddon-moriarty** [comment]
@holynova-SD  No, not yet, sorry. I access it by remote server. I was thinking maybe it had an impact on the ability to get the output back? (not sure why it would though, but I don't see what else could be the problem)

Are you running notebook locally?

**Comment by 41WhiteElephants** [comment]
Probably waveglow is producing ndarray of NaNs. 

to debug try: print(audio[0].data.cpu().numpy())

Change loading model to this, it worked for me: waveglow = torch.hub.load('nvidia/DeepLearningExamples:torchhub', 'nvidia_waveglow')

**Comment by mohamad-hasan-sohan-ajini** [comment]
Hi
Avoid to transfer waveglow weights to fp16 will solve the issue.

regards

**Comment by anaivebird** [comment]
just remove .half() twice on ipynb will solve this problem

**Comment by wsywsywsywsywsy979** [comment]
> just remove .half() twice on ipynb will solve this problem

Hi，inferece code still generates empty outpt(0.00s)  after I remove the .half() twice ,could you give me other advice? (I run the code in docker,I don't know whether it's the main reason.) 
![image](https://user-images.githubusercontent.com/75532068/200244945-4bfcdf74-55e4-4e0a-9a5c-d8080b24e015.png)

**Comment by sun-rabbit** [comment]
hi,I run inference.ipynb  directly on colab，but，But I made this mistake. How can I solve it
<img width="1001" alt="截屏2022-12-14 15 16 44" src="https://user-images.githubusercontent.com/114917342/207530451-db34d3a5-a51a-496f-ab79-d4301ab29a36.png">

**Comment by Nicksooooo** [comment]
> just remove .half() twice on ipynb will solve this problem

Thank you very much.i have succeeded

**Comment by wsywsywsywsywsy979** [comment]
> > just remove .half() twice on ipynb will solve this problem
> 
> Thank you very much.i have succeeded

Thank you, I'm also succeeded. ：）

**Comment by Terrence-zzh** [comment]
> Probably waveglow is producing ndarray of NaNs.
> 
> to debug try: print(audio[0].data.cpu().numpy())
> 
> Change loading model to this, it worked for me: waveglow = torch.hub.load('nvidia/DeepLearningExamples:torchhub', 'nvidia_waveglow')

Genuis！！！
